test/unit/ui/map.test.js

```diff
@@ -1900,20 +1900,20 @@ test('Map', (t) => {
                 // Defaults to style projection
                 t.equal(style.serialize().projection.name, 'globe');
                 t.equal(map.transform.getProjection().name, 'globe');
-                t.equal(map.painter.clearBackgroundTiles.callCount, 0);
+                t.equal(map.painter.clearBackgroundTiles.callCount, 1);
```

Comment:
Are we confident that there's no regression to map load time caused by this additional call to `clearBackgroundTiles`?

Reply:
Just chatted with @karimnaaji about this, benchmarks don't currently implement any non-Mercator projections.

Still, there's probably little to no performance impact here.`clearBackgroundTiles` is a trivial function. https://github.com/mapbox/mapbox-gl-js/blob/5d2b1381b2250633091fef04fe5d681a7795f093/src/render/painter.js#L1027-L1029. The following source cache clearing might be more expensive but might not be an issue if this runs before tiles load.

My previous hypothesis about why this is happening may not be correct since the map seem to be at default zoom 0 here. If there's any easy fix to keep the test the same I think that would be ideal, but this is probably fine.